io/module_wrt_grid_comp.F90

```diff
@@ -754,12 +754,14 @@ subroutine wrt_initialize_p1(wrt_comp, imp_state_write, exp_state_write, clock,
           allocate( wrt_int_state%out_grid_info(n)%latPtr(wrt_int_state%out_grid_info(n)%i_start:wrt_int_state%out_grid_info(n)%i_end, &
                                                           wrt_int_state%out_grid_info(n)%j_start:wrt_int_state%out_grid_info(n)%j_end) )
 
-          do j=wrt_int_state%out_grid_info(n)%j_start, wrt_int_state%out_grid_info(n)%j_end
-          do i=wrt_int_state%out_grid_info(n)%i_start, wrt_int_state%out_grid_info(n)%i_end
-              wrt_int_state%out_grid_info(n)%latPtr(i,j) = latPtr(i,j)
-              wrt_int_state%out_grid_info(n)%lonPtr(i,j) = lonPtr(i,j)
-          enddo
-          enddo
+          if ( trim(output_grid(n)) /= 'regional_latlon_moving' .and. trim(output_grid(n)) /= 'rotated_latlon_moving' ) then
+            do j=wrt_int_state%out_grid_info(n)%j_start, wrt_int_state%out_grid_info(n)%j_end
+            do i=wrt_int_state%out_grid_info(n)%i_start, wrt_int_state%out_grid_info(n)%i_end
+                wrt_int_state%out_grid_info(n)%latPtr(i,j) = latPtr(i,j)
+                wrt_int_state%out_grid_info(n)%lonPtr(i,j) = lonPtr(i,j)
+            enddo
```

Comment:
It's a common practice to indent nested do loops.

Reply:
Done